js/src/tooltip.js

```diff
@@ -415,6 +415,9 @@ const Tooltip = (($) => {
     }
 
     _setListeners() {
+      if (window.Tether === undefined) {
+        throw new Error('Bootstrap tooltips require Tether http://github.hubspot.com/tether/')
```

Comment:
```
if (typeof Tether === undefined) {
```

I'm using webpack provider to provide `window.Tether`, and `Tether` would prefer the latter though.